sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorCustomizer.java

```diff
@@ -62,19 +61,19 @@ public static void customize(AutoConfigurationCustomizer autoConfigurationCustom
         });
         autoConfigurationCustomizer.addSpanExporterCustomizer((spanExporter, configProperties) -> {
             if (spanExporter instanceof AzureMonitorSpanExporterProvider.MarkerSpanExporter) {
-                spanExporter = azureMonitorExporterBuilder.buildSpanExporter(configProperties);
+                spanExporter = azureMonitorExporterBuilder.buildSpanExporter();
             }
             return spanExporter;
         });
         autoConfigurationCustomizer.addMetricExporterCustomizer((metricExporter, configProperties) -> {
             if (metricExporter instanceof AzureMonitorMetricExporterProvider.MarkerMetricExporter) {
-                metricExporter = azureMonitorExporterBuilder.buildMetricExporter(configProperties);
+                metricExporter = azureMonitorExporterBuilder.buildMetricExporter();
```

Comment:
I think we need to pass `configProperties` from above somehow

Reply:
It seems that the previous API was done in this way for testing reasons: https://github.com/Azure/azure-sdk-for-java/blob/f5741c4075e23a6c381e19852bce47c6d97750d3/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java#L321

From a user perspective, it's possible to change the connection string and some stats beat configurations with environment variables (or system properties): 
* https://github.com/Azure/azure-sdk-for-java/blob/f5741c4075e23a6c381e19852bce47c6d97750d3/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java#L73
*  https://github.com/Azure/azure-sdk-for-java/blob/f5741c4075e23a6c381e19852bce47c6d97750d3/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java#L77
* https://github.com/Azure/azure-sdk-for-java/blob/f5741c4075e23a6c381e19852bce47c6d97750d3/sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorExporterBuilder.java#L78

From what I see of the code, `ConfigurationProperties` was added to the build methods only to be able to do this in a certain way.